packages/store/src/decorators/select/symbols.ts

```diff
@@ -0,0 +1,45 @@
+import { Observable } from 'rxjs';
+
+import { CONFIG_MESSAGES, VALIDATION_CODE } from '../../configs/messages.config';
+import { propGetter } from '../../internal/internals';
+import { SelectFactory } from './select-factory';
+import { META_KEY } from '../../symbols';
+
+const DOLLAR_CHAR_CODE = 36;
+
+/**
+ * The SelectUtils namespace is used for logical grouping of functionalities.
+ */
+export namespace SelectUtils {
```

Comment:
Why is this nested in a namespace? Feels like it is trying to be `java` class.
We should just export the functions. This is JavaScript. Functions are first-class citizens.
If there is a good motivation I'm happy to hear it. 👍

Reply:
revert